src/librustc_trans/trans/common.rs

```diff
@@ -429,14 +434,19 @@ impl<'a, 'tcx> FunctionContext<'a, 'tcx> {
 
     pub fn new_block(&'a self,
                      name: &str,
-                     opt_node_id: Option<ast::NodeId>)
+                     opt_node_id: Option<ast::NodeId>,
+                     landing_pad: Option<LandingPad>)
```

Comment:
Most of `new_block` calls pass in `None` for `landing_pad`, it seems. If you’re adding this argument just for MIR, you shouldn’t bother, because the current code which sets the field is just a placeholder and I don’t believe we will be setting this field in MIR on block construction once SEH EH gets implemented (it is quite likely we won’t be setting it at all, even).